xla/service/gpu/gpu_latency_hiding_scheduler.cc

```diff
@@ -448,6 +479,12 @@ int64_t GpuAsyncTracker::GetNumAvailableResources(int64_t resource_type) const {
     return kNumAsyncCollectivesP2P;
   }
 
+  if (resource_type ==
+      ResourceTypeToIndex(GpuResourceType::kGpuAsyncStreamMemcpy)) {
+    constexpr int64_t kNumAsyncMemcpy = std::numeric_limits<int>::max();
```

Comment:
Can you reuse the global const?

Reply:
I think compile-time const like this only used in one place so I agree with TJ that it might be better to define it to the place near its call site.